cmd/orchestrator/main.go

```diff
@@ -98,8 +99,7 @@ func runSetup(ctx context.Context, cli cli, sys *system.System) (*task.Orchestra
 
 	os.Stdin = c.Stdin
 
-	o := task.NewOrchestrator(os.Stdin)
-	sys.AddService(o.Cleanup)
```

Comment:
This was an error before; it should've been added as a cleanup, not a service. Either way, we need to handle cleanup within the `Orchestrator` service now.